site/content/en/docs/Getting started/first_kustomization.md

```diff
@@ -340,43 +301,43 @@ It is important to note here that the name for _both_ the `deployment` and the `
 Moving on to our next requirements, we can set the namespace and the number of replicas we want by using `namespace` and `replicas` respectively:
 
 
-_kustomize-example-2/overlays/production/kustomization.yaml_:
+_kustomize-example/overlays/production/kustomization.yaml_:
 ```yaml
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
-namePrefix: overlay1-
+namePrefix: env1-
 nameSuffix: -production
 
 namespace: production
 
 replicas:
-- name: nginx
+- name: example-nginx
   count: 3
 
 resources:
 - ../../base
 ```
 
-_kustomize-example-2/overlays/staging/kustomization.yaml_:
+_kustomize-example/overlays/staging/kustomization.yaml_:
 ```yaml
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
-namePrefix: overlay2-
+namePrefix: env2-
 nameSuffix: -staging
 
 namespace: staging
 
 replicas:
-- name: nginx
+- name: example-nginx
```

Comment:
@natasha41575 @KnVerey putting just `nginx` works here even though we are updating a resource coming from the base, is this expected?

Reply:
Yep, under the hood, the name reference transformer keeps track of all the names objects have had throughout the build, and as long as there's no ambiguity, it will successfully resolve the names either way.